docs/docs/workflows/verify-cluster.md

```diff
@@ -15,6 +15,15 @@ This command performs the following steps:
 2. Verify the signature of the measurements. This will use Edgeless Systems' [public key](https://edgeless.systems/es.pub).
 3. Write measurements into configuration file.
 
+It's normal that there are more measurements than there are indices in the `enforcedPCRs`.
+This is because only a subset of the available measurements can be verified and reproduced.
+These non-reproducible values typically measure parts of the boot process that are controlled by the CSP.
+You can find a list with descriptions in the [runtime measurements](../architecture/attestation.md#runtime-measurements) section.
+
+You can still set non-reproducible measurements in the `enforcedPCRs` field.
```

Comment:
```suggestion
You can still set non-reproducible measurements in the `measurements` field.
```

Reply:
I am talking about setting a measurement as enforced here. Changed to `enforcedMeasurements`. Not sure how to understand the suggestion?